libraries/chain/include/eosio/chain/block_state.hpp

```diff
@@ -11,7 +11,8 @@ namespace eosio::chain {
       // ------ data members -------------------------------------------------------------
       signed_block_ptr           block;
       bool                       validated;             // We have executed the block's trxs and verified that action merkle root (block id) matches.
-      digest_type                finalizer_digest;
+      digest_type                strong_finalizer_digest;
+      digest_type                weak_finalizer_digest;
```

Comment:
These are not set.

Reply:
I planned to add compute_finalizer_digest in a separate PR. Will put a TODO here for now.

Reply:
For the first prototype, it will be set to block_id in the constructor of block_state when the constructor code is ready.